statsmodels/stats/proportion.py

```diff
@@ -112,14 +112,21 @@ def func(qi):
         ci_low = center - dist
         ci_upp = center + dist
 
+    # This is a typo cleaned up in issue #3167 and will need to be deprecated in
+    # a future release.
     elif method == 'jeffrey':
```

Comment:
replace this by `method[:4] == 'jeff'`
We started to switch to just checking unambigous part of the string in a few methods. This avoids deprecation and more forgiving to users that misspell the name.